packages/GA4Client/src/common.js

```diff
@@ -64,6 +67,29 @@ Common.prototype.truncateEventAttributes = function (eventAttributes) {
     );
 };
 
+Common.prototype.standardizeParameters = function (parameters) {
+    var standardizedParameters = {};
+    Object.keys(parameters).forEach(key => {
+        var standardizedKey = key.replace(/[^a-zA-Z0-9_]/g, '_');
+        for (forbiddenPrefix in FORBIDDEN_PREFIXES) {
+            if (standardizedKey.startsWith(forbiddenPrefix)) {
+                standardizedKey = standardizedKey.replace(forbiddenPrefix.toRegex(), "")
+            }
+        }
+        while (!isEmpty(standardizedKey) && standardizedKey.charAt(0).match(/[^a-zA-Z]/i)) {
```

Comment:
use the created regex variable here

Reply:
this is a different regex that's only used here.